README.md

```diff
@@ -146,6 +163,159 @@ helm_chart(
 ```
 These variables have to be "exported" by the `status.sh` file defined in your project root dir.
 
+### helm_diff
+
+`helm_diff` is used to build and compare the new chart with the currently-running service; this can help determine the actual difference to the running service after making some changes in code: either checking for impact, or confirming that a refactor has an expected level of change to the service.
```

Comment:
```suggestion
`helm_diff` is used to build and compare the new chart with the currently-running service; this can help determine the actual difference to the running service after making some changes in code: either checking for impact, or confirming that a refactor has an expected level of change to the service.
Importantly, although `helm_diff` compares the current code to the running release, it does this by building a dry-run 'style release, and compares that release to the running release.  This is important because there's no static analysis going on, it's generating a release to compare to the running release, not comparing just the chart.
```